src/Adapters/Logger/FileLoggerAdapter.js

```diff
@@ -99,9 +99,12 @@ let _verifyTransports = ({infoLogger, errorLogger, logsFolder}) => {
 }
 
 export class FileLoggerAdapter extends LoggerAdapter {
-  constructor(options = {}) {
+  constructor(options) {
     super();
-
+    if (options && !options.logsFolder) {
+      throw "FileLoggerAdapter requires logsFolder";
```

Comment:
That seems like a weak justification for breaking compatibility to me. I'd suggest documenting that it's required, and warn if it's not supplied, but not actually fail if it's not supplied.